extensions/wikia/Wall/maintenance/CleanupCommentsIndex.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once( getenv( 'MW_INSTALL_PATH' ) !== false ? getenv( 'MW_INSTALL_PATH' ) . '/maintenance/Maintenance.php' : dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+class CleanupCommentsIndex extends Maintenance {
+
+
+	public function execute() {
+		$getIdsQuery = "select page_id from page, comments_index where page_id=comment_id and page_namespace not in (1,500,501,1200,1201,2000,2001)";
+
+		$db = wfGetDB( DB_MASTER );
+
+		$ids = [];
+
+		$result = $db->query( $getIdsQuery );
+		while ( $row = $db->fetchRow( $result ) ) {
+			$ids[] = intval( $row['page_id'] );
+		};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)